include/cinatra/coro_http_client.hpp

```diff
@@ -222,23 +222,16 @@ class coro_http_client {
       }
       else {
         if (has_closed_) {
-          if (!proxy_host_.empty() && !proxy_port_.empty()) {
-            if (ec = co_await asio_util::async_connect(
-                    io_ctx_, socket_, proxy_host_, proxy_port_);
-                ec) {
-              break;
-            }
-          }
-          else {
-            if (ec = co_await asio_util::async_connect(
-                    io_ctx_, socket_, u.get_host(), u.get_port());
-                ec) {
-              break;
-            }
-            // Should the connection be actively closed in proxy? Maybe it's
-            // better to let the user judge.
-            has_closed_ = false;
+          std::string host = proxy_host_.empty() ? u.get_host() : proxy_host_;
+          std::string port = proxy_port_.empty() ? u.get_port() : proxy_port_;
+          if (ec = co_await asio_util::async_connect(io_ctx_, socket_, host,
+                                                     port);
+              ec) {
+            break;
           }
+          // Should the connection be actively closed in proxy? Maybe it's
+          // better to let the user judge.
```

Comment:
the user can call client.close(), is it satisfy the acquirement?

Reply:
Maybe right,I don't know if I should disconnect from the proxy

Reply:
so we can remove the comments.

Reply:
ok

Reply:
finish it.